### DISCUSSION TITLE
QCefView example doesn't initiate under Linux when build type Debug

### DISCUSSION THREAD

**Discussion by EdnY1** [CefView/QCefView#419] [Q&A] [answered]
I'm having this issue where I can't debug under Linux, I tried different distros to make sure and its the same, however release mode works properly. Is there any way to address this issue? am I missing something to be able to debug? 

My changes:
- The only thing I did was changing ``-DCMAKE_BUILD_TYPE=Release`` to ``-DCMAKE_BUILD_TYPE=Debug`` in the ``generate-linux-x86_64.sh``

![image](https://github.com/CefView/QCefView/assets/79129327/2c90d5c8-fcf6-4051-82f9-681d93d637a3)

**Comment by tishion** [comment]
This should be CEF issue, you could try to downgrade CEF version.

**Reply by EdnY1** [reply]
I did try, 118, 115, 113, 110, same behavior, there is something else going on. Do you know if someone was ever able to run debug under Linux?

**Comment by EdnY1** [comment, marked as answer]
Anyone having a similar issue check if you are under wayland compositor, QCefView is currently working only on X11
